app/server/static/components/label.vue

```diff
@@ -52,23 +52,16 @@
                 label.label Color
                 div.field.has-addons
                   div.control
-                    a.button(
-                      v-bind:style="{ \
-                        color: newLabel.text_color, \
-                        backgroundColor: newLabel.background_color \
-                      }"
+                    .form__field
+                        .form__input
+                          swatches(v-model='newLabel.background_color' colors='basic' show-fallback=true popover-to='' :trigger-style="{ width: '36px', height: '36px' }")
```

Comment:
This probably fails puglint since other places in the code separate attributes on the same line with `, `.

Reply:
Addressed